internal/common/config_test.go

```diff
@@ -0,0 +1,13 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewConfig(t *testing.T) {
+	config := NewConfig()
+
+	assert.NotEqual(t, *config, Config{})
```

Comment:
Is this for bypassing the test? this will alway true I think :)

Reply:
ah, honestly, I copied it from other projects :smile: 
I need to come up with a better scenario :eyes:

Reply:
I reworked the test to actually parse the config (I don't fully understand how to mock it)
Wdyt?

Reply:
yep, this is fine, actually we know it partially parse the config.